CoEpi/Dependencies.swift

```diff
@@ -65,14 +68,11 @@ class Dependencies {
     }
 
     private func registerWiring(container: DependencyContainer) {
-        container.register(.singleton) { ScannedCensHandler(coepiRepo: try container.resolve(),
+        container.register(.eagerSingleton) { ScannedCensHandler(coepiRepo: try container.resolve(),
```

Comment:
If eagerSingleton combined with container.bootstrap() instantiates ScannedCensHandler - great :)

Reply:
It does now 🙂

Reply:
It does 🙂